.github/workflows/release.yml

```diff
@@ -5,16 +5,23 @@ on:
     branches: [ master ]
     types:
       - completed
+  workflow_dispatch:
+    inputs:
+      ref:
```

Comment:
Do we need this input? Do we need to run the workflow from one branch's HEAD but release the version from another commit?

Reply:
I think we might want to do this. I'm fine with dropping it now and adding later if needed, though

Reply:
Agreed with Agnieszka to remove it. One can install a requirement from a specific commit, branch, or tag instead.